Logger/BaseEntity.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Logger;
+public abstract record class BaseEntity : IEntity
+{
+    string IEntity.Name { get; } = "";
```

Comment:
```suggestion
    public abstract string Name { get; }
```
This will force deriving classes to implement this